packages/types/src/atoms/graphql-types.ts

```diff
@@ -3083,6 +3117,72 @@ export enum PriorityCode {
   Urgent = 'URGENT'
 }
 
+/** Who, What, When for a set of resources. */
+export type Provenance = {
+  /** Activity that occurred. */
+  activity?: Maybe<CodeableConcept>;
+  /** Actor involved. */
+  agent: ProvenanceAgent;
+  /** An entity used in this activity. */
+  entity?: Maybe<Array<Maybe<ProvenanceEntity>>>;
+  /** Additional content defined by implementations. */
+  extension?: Maybe<Array<Maybe<Extension>>>;
+  /** Logical Id of the resource. */
+  id: Scalars['ID'];
+  /** Flag to state whether the resource should be displayed as entered in error in user interface */
+  isEnteredInError?: Maybe<Scalars['Boolean']>;
```

Comment:
Are we deviating from HL7 a bit with `isEnteredInError`? (entirely possible I'm just looking at the wrong spec)
If so have we checked with Jim this is okay?

Reply:
Good enough for me :)